examples/apps/heart_disease_eval/workload/heart_disease_evaluation_logic.cpp

```diff
@@ -223,17 +222,11 @@ double HeartDiseaseEvalLogic::score_num(int num) {
 // Process work order input for heart disease risk factors
 std::string HeartDiseaseEvalLogic::executeWorkOrder(
         std::string decrypted_user_input_str) {
+    std::string resultString;
     // Variables to accumulate multiple results
     static int totalRisk = 0;
     static int count = 0;
 
-    // Clear accumulation data if input is empty
-    if (decrypted_user_input_str.empty()) {
-        totalRisk = count = 0;
-        return "";
```

Comment:
How are we resetting the totalRisk and count with new logic ? Or is it not required since every new work order request will result in creation of new instance of  work order processor ?

Reply:
I now use `risk` instead of `totalRisk` to calculate the risk of heart disease for one person (line 267).
The original intent of `totalRisk` is for it to be a total of all the risk percentages for all people.  Dividing that by the count gives an average risk: `totalRisk / count`  (line 243).

No, each work order request does not create a new worker instance. That is why I can calculate an average risk (in addition to an individual risk)

Reply:
@manojgop This works good enough for demo purposes using one worker.
But I will add IPC so multiple workers can be used.

Reply:
@danintel . we have only single worker (one enclave) with multiple workload (heart-disease, echo etc). And currently the worker implementation is stateless. For every new work order request , a new instance of workloadProcessor object instance is created.  The scope of static variables (totalRisk,count) is valid only till workloadProcessor object is valid. So even for heart disease eval workload, to calculate aggregate we need pass all input data in single work order request.We can't aggregate the results over multiple work order requests.

Reply:
Looks like value of static variable inside class function is persevered across different instances of class.  I initial thought we have to declare it as static member variable of class to preserve the value across multiple instances.

Reply:
@manojgop The static variable is only shared for a single process. So more work needs to be done to share it across processes--with multiple instances.